lib/fauna/query.rb

```diff
@@ -680,6 +712,20 @@ def self.wrap_values(obj)
       def self.wrap_varargs(values)
         wrap(values.length == 1 ? values[0] : values)
       end
+
+      def self.wrap_write(ref, params)
```

Comment:
I'll just drop the sugar overall, and just require the user to wrap their lambdas themselves with `query`.